src/pages/subreddit/Subreddit.js

```diff
@@ -1,40 +1,48 @@
 import React, { useEffect, useState } from 'react';
 import './Subreddit.css';
 import axios from 'axios';
-import {Link, useHistory} from "react-router-dom";
+import {useParams} from "react-router-dom";
 
 
 function Subreddit() {
-    const history = useHistory();
-    const [subreddit, setSubreddit] = useState('');
+    const [subreddit, setSubreddit] = useState({});
+    const { id } = useParams();
+    // zou ik ook meerdere useParams in kunnen stellen of altijd maar 1?
 
     useEffect(() => {
         async function fetchCategories() {
             try {
-                const response = await axios.get('https://www.reddit.com/r/{id}/about.json');
-                console.log(response.data.data.children);
-                setSubreddit(response.data.data.children);
+                const response = await axios.get(`https://www.reddit.com/r/${id}/about.json`);
+                // Is deze link een speciale link waarbij je de data kunt ophalen via het ID of kun je
```

Comment:
Dit is inderdaad een speciale link ja. Elke api is in die zin verschillend terwijl ze wel een bepaald principe proberen toe te passen. Je zult dus altijd de api moeten raadplegen om te weten hoe je die aan kan roepen en wat je mee moet geven